README.md

```diff
@@ -57,6 +57,21 @@ make manifests
 ```
 Find the manifests in the `_out` directory once done.
 
+### Manifests, CSV generator and HCO integration
+
+The `kubevirt-ssp-operator` provides three way to consume its manifests
+1. individual manifests files under `deploy` and `deploy/crds`. Please note that `deploy/olm-catalog` is autogenerated.
+   these are the authoritative manifests that the developers maintain and enhance, for example when they add features.
+   An end user should not, however, consume them directly.
+   If you want to install the `kubevirt-ssp-operator` *without* HCO, please use the `hack/install-operator.sh` helper.
+2. CSVs and manifests to be used with HCO. HCO is the preferred way to deploy the `kubevirt-ssp-operator`.
+   We provide CSV file and package file (and everything else), autogenerated on release using `make manifests`.
+   This step creates both the unversioned CSV (see below) and the versioned -per release CSV. This is because...
```

Comment:
I would mention where the artifacts can be downloaded from here.

Reply:
done!